README.md

```diff
@@ -116,3 +116,22 @@ Either set `HARMONY_DIR` or replace it with your Harmony project directory path.
 Harmony-py uses [GitHub
 Actions](https://github.com/nasa/harmony-py/actions) to run the Linter
 & Unit Tests. The test coverage output is saved as a build artifact.
+
+## Building and Releasing
+
+If a new version of Harmony-Py has been released then `master` should be tagged with an updated version:
+
+        $ git checkout master
+        $ git tag -a 1.2.3    # where 1.2.3 is the next version number
+
+Then, to generate new package and wheel files:
+
+        $ make build
+
+The Makefile reads the current version number based on git tag, populates the version in `harmony/__init__.py`, and `setup.py` reads the version number from `harmony/__init__.py` for packaging purposes.
+
+Provided API tokens are in order, the following runs the build target and publishes to PyPI:
+
+        # make publish
```

Comment:
If I understand correctly, following these steps will:

- Tag the current branch locally
- Update the contents of `harmony\__init__.py` locally
- Build the package / wheel
- Publish to PyPI

At the end of doing that, there will be an uncommitted change with the correct contents of `harmony/__init__.py` and a _local_ tag with the correct version but not including the change to `harmony/__init__.py`

Thoughts about having `make build` commit the version and `git tag -f` (tag or move an existing tag) the version?  I think keeping it as either a manual git push or putting the push in `make publish` is best so the scary stuff happens in one place.

Reply:
Alternatively, is your thinking that we'd never commit `__init__.py` and always have it some default value in copies from git?  I think that could work, too.

Reply:
There's another I've heard about called versioneer (https://github.com/python-versioneer/python-versioneer) but I've not used it. I think these things are "on the outs" since tools like poetry do this.

Reply:
Based on recent conversations, I guess I would propose the following manual steps and revisit / finish automating in a future ticket. Presumably that ticket would implement a method of versioning and building across all GH hosted projects (or all/most python projects).

In summary:
- `git tag` determines version information
- `make build` stores version info in __init __.py based on tag
- we would commit that change to master, re-apply the tag, then push to master
- `make publish`